src/pages/Animal/Interested/index.tsx

```diff
@@ -155,7 +155,6 @@ const Interested = (): JSX.Element => {
                         onPress: () => navigation.navigate('Chat', {
                           title: user.userName,
                           targetUserUID: u.id,
```

Comment:
A one letter variable name is a little vague... maybe rename this to `interestedUser`?